.github/workflows/release.yml

```diff
@@ -73,18 +73,32 @@ defaults:
 
 jobs:
   bump-version-generate-changelog:
-    name: "Bump package version, Generate changelog"
+    name: "Bump package version, Generate changelog - dbt-adapters"
```

Comment:
Duplicate step to provide different configuration for `dbt-adapters` and `dbt-tests-adapter`. `dbt-tests-adapter` has no tests, and `hatch` needs to be in `dbt-tests-adapter` to reference the correct `pyproject.toml` and associated commands.